build/build.ps1

```diff
@@ -21,12 +21,12 @@ function Build-One {
     );
 
     Write-Host "##[info]Building $project ($action)..."
-    dotnet $action (Join-Path $PSScriptRoot $project) `
-        -c $Env:BUILD_CONFIGURATION `
-        -v $Env:BUILD_VERBOSITY `
-        /property:DefineConstants=$Env:ASSEMBLY_CONSTANTS `
-        /property:Version=$Env:ASSEMBLY_VERSION `
-        /property:QsharpDocsOutputPath=$Env:DOCS_OUTDIR
+    Invoke-Expression ("dotnet $action $(Join-Path $PSScriptRoot $project) " +
+        "-c $Env:BUILD_CONFIGURATION " +
+        "-v $Env:BUILD_VERBOSITY " +
+        "$(if ($Env:ASSEMBLY_CONSTANTS){"/property:DefineConstants=$Env:ASSEMBLY_CONSTANTS"}) " +
```

Comment:
```suggestion
        "$(if ($Env:ASSEMBLY_CONSTANTS) { "/property:DefineConstants=$Env:ASSEMBLY_CONSTANTS" }) " +
```

Reply:
I went ahead and made this an the other formatting changes, as well as ran the autoformatter in VSCode that caught a few more spaces in if-statements that were missing. Thanks!